python/ray/actor.py

```diff
@@ -140,6 +140,58 @@ def __setstate__(self, state):
             hardref=True)
 
 
+class ActorClassMethodMetadata(object):
+    """Metadata for all methods in an actor class. This data can be cached.
```

Comment:
```suggestion
    """Metadata for all methods in an actor class.
```
`This data can be cached.` looks confusing here. We can add comments at the places where we cache this class.

Reply:
Implemented a ActorClassMethodMetadata cache.